src/server/Privilege.ts

```
@@ -0,0 +1,60 @@
+import { PatternDecoder } from "../core/Cosmetics";
+import { Cosmetic } from "../core/CosmeticSchemas";
+type PatternEntry = {
+  pattern: string;
+  role_group?: string[];
+};
+export class PrivilegeChecker {
+
+  constructor(patternData: Cosmetic) {
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)